src/smart-components/order/order-service-form-step-configuration.js

```diff
@@ -3,11 +3,30 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { withRouter } from 'react-router-dom';
 import { Bullseye, Radio, Form, Title, Stack, StackItem } from '@patternfly/react-core';
+import { schemaParsers } from '@data-driven-forms/react-form-renderer';
 
 import FormRenderer from '../common/form-renderer';
 import { fetchServicePlans, sendSubmitOrder } from '../../redux/actions/order-actions';
 import { fetchProviderControlParameters } from '../../helpers/portfolio/portfolio-helper';
 
+/**
+ * TO DO
+ * We need some explicit way of telling what kind of schema is being received
+ */
+const createFormSchema = (servicePlanSchema, providerControlParameters) => {
+  const providerFields = schemaParsers.mozillaParser(providerControlParameters).schema.fields
```

Comment:
If we allow a schema type for the servicePlanSchema, we should do the same for the providerControlParameters , rather than expect these always to be Mozilla type. We could either: 

1. Allow a schema type to be specified for the providerControlParameters 
or
2. Assume the same schema type for the providerControlParameters as the one specified for the servicePlanSchema, with mozzilla as default.

Reply:
I agree we need to get more details about this. That is why i added the TO DO. I expect i'll be revisiting this piece of code very soon.

Reply:
If it helps, I don't think we need the Control Parameters for this initially pass of Tower support.  We can change providerControlParameters to be whatever you need, but the end result is it will not require anything additional to be displayed.

Reply:
Understood - was making a general point, not specific to any type of provider.

Reply:
We can merge this as is for now, and revisit later.